python_backend/api/v1/endpoints/quality.py

```
@@ -180,83 +217,71 @@ async def save_user_feedback(
 @router.post("/company/generate-final", response_model=FinalTextGenerationResponse)
 async def generate_final_integrated_text(
     request: FinalTextGenerationRequest,
-    agent: Annotated[OptimizedEnterpriseQualityAgent, Depends(get_enterprise_quality_agent)]
+    agent: Annotated[object, Depends(get_enterprise_quality_agent)]
```

Ruff (0.13.1)

Unused function argument: `agent`

(ARG001)